### DISCUSSION TITLE
Are you hosting Orchard Core in Azure? We can give you a discount!

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#14185] [General]
Microsoft's Azure cloud is one of the best ways to host an Orchard Core app ([DotNest](https://dotnest.com/) being the best, of course ;)), it also has first-class support built right into Orchard. If you use Azure, now we at Lombiq can provide you with a discount on your Azure usage! We can do this as a Microsoft Partner and reseller of Azure services.

Just write us here: crew@lombiq.com. The level of discount depends on your usage volume and where you are located, but we'll be able to quickly figure it out for you.

**Comment by hishamco** [comment]
Is it a kind of promotion :)

Thanks, Lombiq for all the efforts that you made to Orchard Community, This is a good step that the community might take advantage of, but still sometimes we need the sort of server-side programming with Razor rather than Liquid

**Reply by Piedone** [reply]
Yep, and for those cases the second best hosting option after DotNest remains Azure :D.
